draft-ietf-rats-reference-interaction-models.md

```diff
@@ -196,10 +201,10 @@ Similarly, deviations from the generic model described in this document can be i
 In order to ensure appropriate conveyance of Evidence, the following requirements MUST be fulfilled:
 
 Integrity:
-: Information provided by an Attester MUST NOT have been altered since it was created. This may be achieved through a digital signature over Attestation Evidencewhich may be symmetric, like an HMAC, or asymmetric, like ECDSA.
+: Information provided by an Attester MUST NOT have been altered since it was created. This may be achieved through by symmetric signature, e.g., PSA TF-M profile, or asymmetric, like ECDSA.
```

Comment:
```suggestion
: Information provided by an Attester MUST NOT have been altered since it was created. This may be achieved via symmetric cryptography, e.g., using COSE Mac0 as in PSA TF-M profile ({{Section 5.2 of I-D.tschofenig-rats-psa-token}}), or asymmetric, like ECDSA.
```